Pod/Classes/Data.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  Data.swift
+//  Pods
+//
+//  Created by Adam Galloway on 10/7/15.
+//
+//
+
+import Foundation
+
+public final class Data: Model,ResponseObject,ResponseCollection {
+
+    public let group: String
```

Comment:
we can change this for public var group: String = "" so we don't need the else condition